src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { ErrorType } from '../../types/error-enum';
+import { PersonInfo } from '../PersonInfo/PersonInfo';
+
+export const PeoplePage = () => {
+  const [peopleFS, setPeopleFS] = useState<Person[] | null>(null);
+  const [error, setError] = useState<string>('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  const hasNoPeopleFromSrver = peopleFS?.length === 0;
+  const hasLoadingError = error === ErrorType.PEOPLE_LOADING;
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(people => {
+        setPeopleFS(people);
+      })
```

Comment:
```suggestion
      .then(setPeopleFS)
```
You can simplify in this case